ember/Dockerfile

```diff
@@ -14,6 +14,8 @@ ENV EMBER_GIT_BRANCH=${EMBER_GIT_BRANCH:-master} \
     FEDORA_ADAPTER_USER="" \
     USER_SERVICE_URL=/pass-user-service/whoami \
     SCHEMA_SERVICE_URL=/schemaservice \
+    DOI_SERVICE_URL=/doiservice/journal \
+    POLICY_SERVICE_URL=/policies \
```

Comment:
When running `pass-docker` I get the following requested URL:

```
https://pass.local/policies/policies?submission=https://pass.local/fcrepo/rest/submissions/83/b6/cb/22/83b6cb22-e3d5-4bfe-83b6-55083f9caf93
```

The submission ID doesn't matter so much, but you can see that the relative URL "worked" but it picked the default value from this Dockerfile, instead of the correct value from `.env`. I'm not exactly sure but should this be something like `POLICY_SERVICE_URL=${POLICY_SERVICE_URL:-/policyservice}` or just `=/policyservice`. (I sort of forget how to pull in vars from `.env` while also having a default value)